<?xml version="1.0"?>
<launch>
  <!-- <rosparam command="load" file="$(find beb)/src/bebop2.yaml" />  -->
  <arg name="frame" default = "Parrot_umar"/>
  <arg name="worldFrame" default="world"/>
  <arg name="radius" default="0.3"/>
  <arg name="trajectory" default="circle"/>
 
    <!--   <node name="PlotOps" pkg="beb" type="PlotOps.py" output="screen">
    </node> -->
    <node name="TrainOps" pkg="beb" type="TrainingOps.py" output="screen">
    </node> 

      <!--


      <node name="random_bebop_commands" pkg="beb" type="beb" output="screen">

    <param name="frame" value="$(arg frame)" />
    <param name="worldFrame" value="$(arg worldFrame)" />
    <rosparam command="load" file="$(find beb)/src/bebop2.yaml" />
  
  </node>
  
   <node name="talker" pkg="beb" type="MPC.py" output="screen">
     <param name="radius" value="$(arg radius)" />
    </node>

    <node name="plotter" pkg="beb" type="plotter.py" output="screen">
      <param name="frame" value="$(arg frame)" />
    </node>

    <node name="Optimizer" pkg="beb" type="Optimizer.py" output="screen">
      <param name="frame" value="$(arg frame)" />
    </node>

    <node name="BagOperations" pkg="beb" type="Bagops.py" output="screen">
      <param name="frame" value="$(arg frame)" />
      <param name="radius" value="$(arg radius)" />
      <param name="trajectory" value="$(arg trajectory)" />
    </node>
    

     <include file="$(find qualisys)/launch/qualisys.launch">
     </include>

    <include file="$(find qualisys)/launch/qualisys_odom.launch">
        <arg name="model" default="$(arg frame)"/>
        <arg name="output" default="screen"/>
    </include>
  -->

</launch>